Ui/DataProvider/Subscription/Form/Modifier/PaymentProfile.php

```diff
@@ -88,11 +88,15 @@ public function modifyMeta(array $meta)
     public function getPaymentProfiles()
     {
         $options = [];
-        $customer = $this->getCustomerId();
-        $request = $this->api->request('payment_profiles?query=customer_id%3D' . $customer . '&status=active', 'GET');
+
+        $endpoint = 'payment_profiles?query=customer_id%3D' . $this->getCustomerId() . '%20status%3Dactive';
```

Comment:
Nesse caso é uma questão de segurança especificar o tipo do perfil de pagamento, que atualmente é exclusivamente **Cartão de crédito**.
O que acha? :thought_balloon: 
```suggestion
        $endpoint = 'payment_profiles?query=customer_id%3D' . $this->getCustomerId() . '%20status%3Dactive%20type%3DPaymentProfile%3A%3ACreditCard';
```